README.md

```diff
@@ -211,14 +204,35 @@ ANSIBLE_COLLECTIONS_PATHS=~/my_dev_path ansible-playbook -i inventory.network te
 
 ## Release notes
 
-Release notes are available [here](https://github.com/ansible-collections/vyos.vyos/blob/main/CHANGELOG.rst).
+Release notes are available [here](https://github.com/vyos/vyos.vyos/blob/main/CHANGELOG.rst).
 
 ## Roadmap
 
 <!-- Optional. Include the roadmap for this collection, and the proposed release/versioning strategy so users can anticipate the upgrade/update cycle. -->
 
+Major Version | Ansible Support | VyOS Support | Details
+--- | --- | --- | ---
+4.1.0 | 2.15 | 1.1.2 | Final release for the 4.x series
+5.0.0 | 2.16 | 1.1.2 | First relase under VyOS control as a separate collection
+6.0.0 | 2.18 | 1.3.8 | *Planned* release for supporting VyOS 1.3.8+
+7.0.0 | x.xx | 1.4.x | *Prospective* release deprecating incompatible 1.3.x modules
```

Comment:
@andamasov I made some changes to the README in this PR to bring it more in line with VyOS ownership. I've also added a very vague version-based roadmap here in order to clarify that 6.0 is the 1.3.8+ release and 7.0 is the first release that may not support 1.3 (if incompatible in some fashion we can't or won't).